app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/moviedetails/view/fragment/MovieDetailsFragment.kt

```diff
@@ -8,21 +8,27 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import com.example.marcinwisniewski.intivetrainingapp.R
-import com.example.marcinwisniewski.intivetrainingapp.databinding.DetailsFragmentBinding
-import com.example.marcinwisniewski.intivetrainingapp.moviedetails.viewmodel.DetailsViewModel
+import com.example.marcinwisniewski.intivetrainingapp.databinding.MovieDetailsFragmentBinding
+import com.example.marcinwisniewski.intivetrainingapp.moviedetails.viewmodel.MovieDetailsViewModel
 import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
 
-class DetailsFragment : Fragment() {
-    private lateinit var viewModel: DetailsViewModel
+class MovieDetailsFragment : Fragment() {
+
+    private lateinit var movieDetailsViewModel: MovieDetailsViewModel
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        val detailsBinding: DetailsFragmentBinding = DataBindingUtil.inflate(layoutInflater, R.layout.details_fragment, container, false)
-        fetchCurrentMovie()
-        detailsBinding.viewModel = viewModel
+        val detailsBinding: MovieDetailsFragmentBinding = DataBindingUtil.inflate(layoutInflater, R.layout.movie_details_fragment, container, false)
+        movieDetailsViewModel = ViewModelProviders.of(this).get(MovieDetailsViewModel::class.java)
```

Comment:
Add TODO how to provide view model via DI

Reply:
added